.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    branches: [main] # run on PRs opened against `main`
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        node: [16]
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Yarn Install
+        run: yarn install --frozen-lockfile
+
+      - name: TypeScript Compilation
+        run: yarn tsc --noEmit
```

Comment:
Add a script in `package.json` that runs type check. Something like `yarn type-check`.

Reply:
Fixed.